app/controllers/no_match_controller.rb

```diff
@@ -13,9 +13,25 @@ def create
     if @no_match_form.valid? && @no_match_form.try_again?
       redirect_to check_answers_path
     elsif @no_match_form.valid?
-      create_zendesk_ticket
+      if trn_request.from_get_an_identity?
+        begin
+          # Send a request to Get An Identity API with no TRN found
+          IdentityApi.submit_trn!(trn_request, session[:identity_journey_id])
 
-      redirect_to helpdesk_request_submitted_path
+          # Send the user back to Get an Identity
+          redirect_to session[:identity_redirect_uri], allow_other_host: true
+        rescue IdentityApi::ApiError,
+               Faraday::ConnectionFailed,
+               Faraday::TimeoutError,
+               ActionController::ActionControllerError
+          # Do not redirect back to Get An Identity
+          render "errors/unprocessable_entity", status: :unprocessable_entity
```

Comment:
I don't know if 422 is correct here; unprocessable entity means that the request was understood, is well formed, but the server couldn't handle it *and it is the client's fault in some way.* If the downstream API fails, it isn't the client's fault. It also generally means that the client should not repeat the request without any changes, as it'll just be another 422, which isn't the case here.

Probably more of a 500 scenario + a sentry alert.